src/bento/.eslintrc.js

```diff
@@ -0,0 +1,13 @@
+// These extensions have been modified in the last 2 weeks, so we're not
```

Comment:
Curious about how we plan to maintain this file long term given that this list can get outdated pretty quickly. Okay to address separately.

Reply:
I've held the opinion for a long time that we should get rid of the file, update everything, make folks deal with merge conflicts. cc @rcebulko

Reply:
I support this as well, teams have had plenty of time to adjust at this point

Reply:
Note: I removed this file

Reply:
That's awesome news! I'll go ahead and delete this file then.